src/TesterInternal/Config_Client0.xml

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
no need for all those different files. Instead use programmable APIs to tweak configuration (Look for AdjustConfiguration).